src/Instrumentation/Yii/composer.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "open-telemetry/opentelemetry-auto-yii",
+    "description": "OpenTelemetry auto-instrumentation for Yii",
+    "keywords": ["opentelemetry", "otel", "open-telemetry", "tracing", "yii", "instrumentation"],
+    "type": "library",
+    "homepage": "https://opentelemetry.io/docs/php",
+    "readme": "./README.md",
+    "license": "Apache-2.0",
+    "minimum-stability": "dev",
+    "require": {
+      "php": "^8.0",
+      "ext-opentelemetry": "*",
+      "yiisoft/yii2": "^2.0.13",
+      "open-telemetry/api": "^1.0.0beta10",
```

Comment:
this can be `^1` now that we've released GA (and the minimum-stability can be removed, too, I think)

Reply:
Changed. Also removed minimum-stability.